cmd/hack_push_flag.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/spf13/cobra"
+)
+
+var hackPushFlagCommand = &cobra.Command{
+	Use:   "hack-push-flag [(true | false)]",
+	Short: "Displays or sets your hack push flag",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println(strconv.FormatBool(git.ShouldHackPush()))
+			return
+		}
+
+		git.UpdateShouldHackPush(args[0] == "true")
```

Comment:
Move this into an `setHackPushFlag(bool)` method in this file. It won't save space, but it makes the code base more self-describing. The `Run` method of commands is kind of like a controller in web servers. It should not execute any business logic itself. It should only be concerned with gathering data from the peripherals, and then call into a business logic layer that does the actual things.